app/views/coronavirus_landing_page/components/shared/_related_topic_section.html.erb

```diff
@@ -0,0 +1,38 @@
+<%
+  tracking_category ||= nil
+  tracking_action ||= nil
+  tracking = true if tracking_category.present? && tracking_action.present?
+%>
+
+<section class="covid__topic-wrapper">
+  <%= render "govuk_publishing_components/components/heading", {
+    text: related_topic_section["header"],
+    padding: true,
+    border_top: 2,
+    margin_bottom: 6
+  } %>
+
+  <ul class="govuk-list">
+    <% related_topic_section["links"].each do | link | %>
+      <li class="covid__topic-list-item">
+        <%
+          data = if tracking
+            {
+              module: "gem-track-click",
+              track_category: tracking_category,
+              track_action: tracking_action,
+              track_label: link["url"],
+            }
+          end
+        %>
+
+        <%= link_to(
+          link["label"].html_safe,
```

Comment:
Are we expecting HTML to be present here? It doesn't seem too safe to do this and the screenshots don't look like we'd need it.

Reply:
Good spot - will remove.  Thanks